Init demo-app

tb opened the pull request:
I have made initial setup of demo-app.

There are a couple unresolved issues throught:
- had to comment out moduleId: module.id
- had to use full path to import variables / mixins

tb commented:
import variables / mixins, through it could be fixed for webpack with:

    @import "~angular2-mdl/src/scss-mdl/variables";
    @import "~angular2-mdl/src/scss-mdl/mixins";

mseemann commented:
Hi tb!

You are too fast ;-)

Just working on the demo/e2e app. But it seems that i can merge your work without problems because you have added a root folder "demo-app". Do you think your pr is ready for a merge? Otherwise I need to push my work first and you need to rebase with some small conflicts...

tb commented:
I have updated the PR, refactored the isVisible into 

    @HostBinding('class.is-visible') public isVisible = false;

I was not able to fix test through.

I think you can merge it as it gets us into right direction.
